### ISSUE TITLE
Preload default images 

### ISSUE THREAD

**Issue by r2d4** [kubernetes/minikube#1660]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Feature request

* Decrease startup time by preloading the default images into minikube
* Minikube usable offline with no additional setup (provided you have downloaded minikube and the iso already)
```
gcr.io/google_containers/k8s-dns-sidecar-amd64         1.14.4              38bac66034a6        3 days ago          41.81 MB
gcr.io/google_containers/k8s-dns-kube-dns-amd64        1.14.4              a8e00546bcf3        3 days ago          49.38 MB
gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64   1.14.4              f7f45b9cb733        3 days ago          41.41 MB
gcr.io/google-containers/kube-addon-manager            v6.4-beta.2         0a951668696f        2 weeks ago         79.23 MB
gcr.io/google_containers/kubernetes-dashboard-amd64    v1.6.1              71dfe833ce74        6 weeks ago         134.4 MB
gcr.io/google_containers/pause-amd64                   3.0                 99e59f495ffa        14 months ago       746.9 kB
```
Upper bound on total image size to cache (no compression/no shared layers):  347 MB
Current ISO Size: 91 MB

We could add a systemd unit that depends on docker.service similar to minikube-automount.  It would untar an image bundle and then `docker load` each image.

**Comment by r2d4** [comment]
This might also be beneficial to @aaron-prindle as it might make the startup time more reliable (since it won't be dependent on the network)

**Comment by dlorenc** [comment]
Something like this sgtm. I think docker load can actually save multiple images at once, which might help it take advantage of shared layers if any exist.

**Comment by aaron-prindle** [comment]
SGTM.  I assume that We should probably also release the tars on our release page so that that users can download all of the components to run minikube w/o internet in the future.

**Comment by budhrg** [comment]
Any idea on the implementation details? 

We recently implemented similar feature in minishift [here](https://github.com/minishift/minishift/pull/1017). 
Once the provisioning is done, we are running a background process to export the container images to minishift homedir.

**Comment by r2d4** [comment]
@budhrg See #1671 for my first draft

**Comment by r2d4** [comment]
Fixed by #1881
